frontend/lib/norent/faqs.tsx

```diff
@@ -105,10 +106,20 @@ export const NorentFaqsPage: React.FC<{}> = () => {
               const faqs = FaqsContent.filter(
                 (faq) => faq.category === category
               );
+
+              const formatCategoryID = function (
+                categoryTitle: string
+              ): string {
+                return categoryTitle.replace(/\s+/g, "_").toLowerCase();
+              };
```

Comment:
Ooo this is the ideal kind of thing to make a unit test for because (A) it's really easy and (B) it helps document the behavior of the code (it's very hard to tell from reading it what it's doing, but looking at a test helps a lot) and (C) it will make sure there aren't any weird boogs.

Reply:
great! added it to that issue